src/storage-types/src/sources/load_generator.rs

```diff
@@ -376,6 +399,7 @@ impl LoadGenerator {
             LoadGenerator::Marketing => false,
             LoadGenerator::Datums => true,
             LoadGenerator::Tpch { .. } => false,
+            LoadGenerator::KeyValue(_) => false,
```

Comment:
The key value generator is monotonic, no?

Reply:
ah yes!